src/auro-flight-main.js

```diff
@@ -42,11 +46,55 @@ class AuroFlightMain extends LitElement {
     `;
   }
 
+  connectedCallback() {
+    super.connectedCallback();
+
+    /**
+     * Time template object used by convertTime() method.
+     */
+    this.timeTemplate = {
+      hour: "2-digit",
+      minute: "2-digit",
+      timeZone: this.timeZone
+    };
+
+    this.template = {};
+  }
```

Comment:
Not sure why this is in connectedCallback - I think the constructor would be a better place.